models/DataObject/ClassDefinition/DynamicOptionsProvider/MultiSelectOptionsProviderInterface.php

```diff
@@ -18,6 +18,9 @@
 
 use Pimcore\Model\DataObject\ClassDefinition\Data;
 
+/**
+ * @deprecated use SelectOptionsProviderInterface instead
+ */
```

Comment:
I think we should also trigger a deprecation when someone uses this interface.

```suggestion
trigger_deprecation('pimcore/pimcore', '11.1', '%s is deprecated. Use %s instead.', MultiSelectOptionsProviderInterface::class, SelectOptionsProviderInterface::class);

/**
 * @deprecated use SelectOptionsProviderInterface instead
 */
```